native/src/linux/package.json

```diff
@@ -1,11 +1,14 @@
 {
   "name": "@fluvio/native-linux",
-  "version": "0.1.0-alpha.3",
+  "version": "0.1.0-alpha.5",
```

Comment:
how do we update the version next time?

Reply:
There are now separate workflows for building and publishing the native platform specific modules; another for the @fluvio/native that likely will not change much, and then the @fluvio/client which will only change if there are changes to the interface.

The platform modules need to be published before the client, if changes are made.

I'll write some documentation for developing tomorrow. For now, the publishing has been run manually, I am going to remove the publish on tags for the @fluvio/native and platform specific modules. I think it is is easier to sequence these manually for now.

Reply:
commit 58d0362 changes so only the @fluvio/client module will be published on tagged release; The other modules will need to be manually dispatched when changes are made.